packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -497,7 +497,9 @@ export const startVaultFactory = async (
   ]);
 
   const voteCreator = Far('vaultFactory vote creator', {
-    voteOnParamChanges: E(governorCreatorFacet).voteOnParamChanges,
+    /** @type {VoteOnParamChanges} */
+    voteOnParamChanges: (...args) =>
```

Comment:
this makes me think it might be worth having a helper to bind a function and keep its type signature

Reply:
What would such a helper look like?

Reply:
_waves hand_… ??
```
    voteOnParamChanges: bindHere(E(governorCreatorFacet).voteOnParamChanges))
```
or maybe something more targeted,
```
    voteOnParamChanges: E.delegate(governorCreatorFacet).voteOnParamChanges,
```

Either way we could define the types so the signature doesn't have to be repeated.

Reply:
How would `bindHere` or `E.delegate` obtain the `governorCreatorFacet`? It would need that in order to do the binding. Are you thinking of a hidden coordination between `E` and (`bindHere` or `E.delegate`)? Not out of the question, but seems like too much magic to me. I'd rather see the manual binding, I think. But perhaps examples would make a compelling case?

Reply:
I haven't thought it through. If I find time to I'll back with examples